### PR TITLE
First and Second modules completed

### PR DESCRIPTION
**Pull request by MariaShymko**
@szikszail Could you please review my task? Thank you!

### CODE REVIEW

File: module-1/classification.js [hunk 1]
```diff
@@ -23,7 +23,15 @@ function grade(score) {
...
     // PLACE YOUR CODE BETWEEN THIS...
-
+	if (score < 0 || score > 100) {
+        gradeOfStudent = 0;
+    } else if (score >= 90) {
+		gradeOfStudent = 5;
+	} else if (score <= 59 ) {
+		gradeOfStudent = 1;
+	} else {
+	    gradeOfStudent = Math.floor(0.1*score - 4);
```
Comment (szikszail) [inline module-1/classification.js]: If you continue the thread you started here, you can easily make a 1 line mathematical expression.

**Review by szikszail — CHANGES_REQUESTED** [review]
Please check my comment. I'd be happy to review your other solutions.

File: module-1/classification.js (hunk 1 above)
Comment (MariaShymko) [reply module-1/classification.js]: @szikszail I've added a new solution, please review, I hope you will like it 😊 (I was trying not to repeat existing solutions and still make it 1 line mathematical expression)

File: module-1/classification.js [hunk 2]
```diff
@@ -23,7 +23,11 @@ function grade(score) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    gradeOfStudent = 
+	    Math.pow(
```
Comment (szikszail) [inline module-1/classification.js:27]: Nice! Please ping me on Skype4Business

**Review by szikszail — COMMENTED** [review]
Nice solution. I am waiting for your next implementations.

File: module-1/classification.js (hunk 2 above)
Comment (MariaShymko) [reply module-1/classification.js:27]: @szikszail Could you please review the solution for the second module? Thank you!

File: module-2/test/calc.spec.js [hunk 3]
```diff
@@ -4,20 +4,140 @@ const calc = require('../calc');
...
+	 * calc(10).modulo(5).v // 0
+	 * calc(5).divide(0) // throw error
+	 * calc(-3).sqrt() // throw error
+	 * calc(3).add(4)
+	 *     .minus(3)
+	 *     .times(6).v // 24
+	 */
+	// TODO: write test cases to test calculator
 
+	it("should have proper value",
```
Comment (szikszail) [inline module-2/test/calc.spec.js:23]: Quite interesting formatting :)

**Review by szikszail — COMMENTED** [review]
It would be good to see your solutions for the rest of the tasks :)

**Comment by szikszail** [comment]
Thank you for your solution!

**Pull request closed without merging**